addons/netfox/rollback/rollback-synchronizer.gd

```diff
@@ -176,7 +176,17 @@ func _after_tick(_delta, _tick):
 	if not _auth_input_props.is_empty():
 		var input = PropertySnapshot.extract(_auth_input_props)
 		_inputs[NetworkTime.tick] = input
-		rpc("_submit_input", input, NetworkTime.tick)
+
+		#Send the last n inputs for each property 
+		var inputs = {}
+		for i in range(0, NetworkRollback.input_redundency):
+			var tick_input = _inputs.get(NetworkTime.tick - i, {})
+			for property in tick_input:
```

Comment:
Why not send an array of inputs ( i.e. dicts )?

Reply:
I was thinking it would be less data to transmit this way if I'm understanding correctly?

{ 'motion': [(0,1),(0,0),(0,1)], 'jump': [0,0,1] ) }

vs

[{'motion': (0,1) 'jump': 0)}, {'motion': (0,0) 'jump': 0)},{'motion': (0,1) 'jump': 1)}]

Reply:
Fair enough!